randovania/games/samus_returns/pickup_database/pickup-database.json

```diff
@@ -219,7 +228,9 @@
             "model_name": "powerup_missilelauncher",
             "offworld_models": {
                 "am2r": "sItemMissileLauncher",
-                "dread": "powerup_missile"
+                "dread": "powerup_missile",
+                "prime1": "Missile",
+                "prime2": "MissileLauncher"
```

Comment:
why not use what dread does here with MissileExpansionPrime1

Reply:
What's the difference? (also I think I copied from am2r but im not sure)

Reply:
it'll look like the prime 1 missile and thus make it more unique.